android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -398,15 +430,25 @@ private void setText(View view, int textViewId, String text) {
     @SuppressLint("InflateParams")
     @Override
     public View getView(int i, View view, ViewGroup viewGroup) {
-      // Get the list view item for the given position
-      if (view == null) {
-        view = getActivity().getLayoutInflater().inflate(R.layout.list_item_nearby_beacon,
-                                                         viewGroup, false);
-      }
-
       // Display the pwsResult.
       PwPair pwPair = getItem(i);
       PwsResult pwsResult = pwPair.getPwsResult();
+      if (isFolderItem(pwPair)) {
+        view = getActivity().getLayoutInflater().inflate(R.layout.folder_item_nearby_beacon,
+            viewGroup, false);
+        WifiManager wifiManager = (WifiManager) getActivity().
+            getSystemService(Context.WIFI_SERVICE);
+        WifiInfo wifiInfo = wifiManager.getConnectionInfo();
+        String ssid = wifiInfo.getSSID().trim();
+        if (ssid.charAt(0) == '"' && ssid.charAt(ssid.length()-1) == '"') {
+          setText(view, R.id.title, ssid.substring(1,ssid.length() -1));
```

Comment:
Add spaces before and after arithmetic operators and after commas.
```
        if (ssid.charAt(0) == '"' && ssid.charAt(ssid.length() - 1) == '"') {
          setText(view, R.id.title, ssid.substring(1, ssid.length() - 1));
```
LGTM otherwise